dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+                    .setSQL(String.format("select drawed_body, body from template where inode = '%s'", template.getInode()))
+                    .loadResults();
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with absolute path container
+     * Should: Should does not anything
+     */
+    @Test
+    public void whenTemplateLayoutHasAbsolutePathShouldKeepThePathWithoutChange() throws IOException, DotDataException, DotSecurityException {
+
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(jsonDrawBody, "//" + host.getHostname());
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, "//" + host.getHostname()))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with relative path container
+     * Should: Should change the relative path by absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasRelativePathShouldTurnIntoAbsolutePath() throws DotDataException {
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(legacyHTMLLayout, "");
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, ""))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+        CacheLocator.getTemplateCache().remove(template.getInode());
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with absolute path container
+     * Should: Should keep the absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasAbsolutePath() throws DotDataException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)